src/graph.ts

```diff
@@ -130,6 +161,7 @@ export class FactoryGraph {
      */
     containers = new Set<ContainerNode>()
     industries = new Set<IndustryNode>()
+    outputs = new Set<ConsumerNode>()
```

Comment:
This should probably be a `OutputNode`.
```suggestion
    outputs = new Set<OutputNode>()
```

Reply:
Fixed in https://github.com/lgfrbcsgo/du-factory-generator/commit/2f4a32d097a7f59ef06a5d257c5e7cee14db819e